games-action/dxx-rebirth/metadata.xml

```diff
@@ -20,16 +20,24 @@
 		DXX-Rebirth.
 	</longdescription>
 	<use>
-		<flag name="data">Install data files</flag>
-		<flag name="descent1">Install Descent 1 resources and executable</flag>
-		<flag name="descent2">Install Descent 2 resources and executable</flag>
+		<flag name="d1x">Install Descent 1 resources and executable</flag>
+		<flag name="d2x">Install Descent 2 resources and executable</flag>
+		<flag name="descent1">Install Descent 1 resources and executable (used prior to version 0.61)</flag>
+		<flag name="descent2">Install Descent 2 resources and executable (used prior to version 0.61)</flag>
+		<flag name="data">Install the playable demo data (i.e. shareware levels)</flag>
```

Comment:
USE=data causes an any-of dependency on game data packages.  Demo data is one way to satisfy that dependency, but if the user has already installed the package that represents retail data, then the retail data package will satisfy USE=data and the demo data will not be installed.

Reply:
Fair point, I will update this.